packages/roosterjs-editor-types/lib/interface/ImageEditOptions.ts

```diff
@@ -46,4 +46,16 @@ export default interface ImageEditOptions {
      * @default A predefined SVG icon
      */
     rotateIconHTML?: string;
+
+    /**
+     * Adapt handles number to the image size
+     * 8 handlers: 100x100px or bigger
+     * 4 handlers: smaller than 100x100px
+     */
+    sizeAdaptiveHandles?: boolean;
+
+    /**
+     * The handlers should have circular borders
+     */
+    circularHandles?: boolean;
```

Comment:
Instead of having a parameter here, I prefer using editor ExperimentalFeature. So once we feel it is good enough we can turn it on by default